app/models.py

```diff
@@ -72,6 +70,11 @@ def find_user_by_id(id):
         }
         return result
 
+@login.user_loader
+def load_user(id):
+    return User.query.get(int(id))
```

Comment:
If you inherrit basemodel inside your user model, then you can define a find_by_id method inside base model and you can make your method like this:

```
return User.find_by_id(id)
```